tests/dummy/config/environment.js

```diff
@@ -1,15 +1,21 @@
-/* jshint node: true */
+'use strict';
 
 module.exports = function(environment) {
-  var ENV = {
+  let ENV = {
     modulePrefix: 'dummy',
-    environment: environment,
-    baseURL: '/',
+    environment,
+    rootURL: '/',
     locationType: 'auto',
     EmberENV: {
+      _ENABLE_LEGACY_VIEW_SUPPORT: true,
```

Comment:
I think we should include (or not) the `ember-legacy-views` addon in order to test this (vs hard coding this here).